src/sagas.js

```diff
@@ -697,13 +697,19 @@ function adjustVVFile ({ data, options, usbFilter }) {
   if (usbFilter) {
     data = data.replace(/^\[virt-viewer\]$/mg, `[virt-viewer]\nusb-filter=${usbFilter}`)
   }
+  if (options && isSpice) {
+    const smartcardEnabled = options.get ? options.get('smartcardEnabled') : options.smartcardEnabled
```

Comment:
AFAIK `adjustVVFile` function is only called from one place and param `options` can only originate from one place. How comes that we can't be sure whether it's a regular object or a immutable collection? Or is this "checking get method" pattern about something else?

Reply:
Because `options` may comes from two different places: `Selectors.getConsoleOptions` and saga worker `getConsoleOptions`.